counting/script.js

```diff
@@ -0,0 +1,36 @@
+for (let i = 1; i < 11; i++){
```

Comment:
I would prefer to see the condition written as 'i <= 10'. Just feels a little more readable when we want the loop to end at 10. This is mostly just a nitpick.